components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md

```diff
@@ -127,11 +127,11 @@ Set the above prerequisites in `~/.aws/credentials` in Bastion Host:
       - Set `chef_ebs_volume_iops`, `chef_ebs_volume_size` based on your load needs.
       - Set `automate_ebs_volume_type`, `chef_ebs_volume_type`. Default value is `"gp3"`. Change this based on your needs.
 
-{{< warning >}}
-{{% automate/char-warn %}}
-{{< /warning >}}
+    {{< warning >}}
```

Comment:
Try this:

```suggestion
    {{< warning space=4 >}}
```

This will fix the problem @Dmaddu  pointed out in https://github.com/chef/automate/pull/7717/files#r1101869936

Reply:
Thank you @IanMadd